_includes/body-landing.html

```diff
@@ -1,4 +1,4 @@
-<body ng-app="Docker" ng-controller="DockerController" id="landing">
+<body ng-app="Docker" ng-controller="DockerController" id="landing" class="landing">
```

Comment:
I think the id, and the `ng-` attributes can be removed now, but will look separately at those.